events/models.py

```diff
@@ -520,6 +524,31 @@ class Meta:
     def __str__(self):
         return f"{self.pk} {self.child_id}"
 
+    def _send_enrolment_creation_notification(self):
+        attachments = []
+        if settings.KUKKUU_TICKET_VERIFICATION_URL:
+            ticket_qrcode_content = settings.KUKKUU_TICKET_VERIFICATION_URL.format(
+                reference_id=self.reference_id
+            )
+            ticket_qrcode = create_qrcode(
+                ticket_qrcode_content,
+                "svg",
+            )
+            attachments.append(
+                (
+                    f"ticket-{self.reference_id}.svg",  # file name
+                    ticket_qrcode.decode(),  # content (decoded)
+                    "image/svg+xml",  # MIME-type
+                )
+            )
```

Comment:
Haven't been able to fully test this yet, because the mails does not appear in my inbox. Not sure should the QR-code be a SVG or a PNG, decoded or not.

Reply:
Moved this to a new notification service